local-interchaintest/examples/two_party_pol_astroport_neutron.rs

```diff
@@ -0,0 +1,1052 @@
+                    info: AssetInfo::NativeToken {
+                        denom: NTRN_DENOM.to_string(),
+                    },
+                    amount: Uint128::new(ntrn_deposit),
+                },
+                Asset {
+                    info: AssetInfo::NativeToken {
+                        denom: token.clone(),
+                    },
+                    amount: Uint128::new(token_deposit),
+                },
+            ],
+            slippage_tolerance: None,
+            auto_stake: None,
+            receiver: None,
+            min_lp_to_receive: None,
+        };
+    contract_execute(
+        test_ctx
+            .get_request_builder()
+            .get_request_builder(NEUTRON_CHAIN_NAME),
+        pool_addr,
+        DEFAULT_KEY,
+        &serde_json::to_string(&provide_liquidity_msg).unwrap(),
+        &format!(
+            "--amount {}{},{}{} {}",
+            token_deposit,
+            token.clone(),
+            ntrn_deposit,
+            NTRN_DENOM,
+            GAS_FLAGS
+        ),
+    )
+    .unwrap();
+    std::thread::sleep(std::time::Duration::from_secs(3));
+
+    info!("Set up the workflow manager...");
+    let mut workflow_config = WorkflowConfig {
+        owner: NEUTRON_CHAIN_ADMIN_ADDR.to_string(),
+        ..Default::default()
+    };
+    let neutron_domain =
+        valence_workflow_manager::domain::Domain::CosmosCosmwasm(NEUTRON_CHAIN_NAME.to_string());
+
+    // We will need 10 base accounts
+    for i in 1..11 {
+        workflow_config.accounts.insert(
+            i,
+            AccountInfo {
+                name: format!("base_account_{}", i),
+                ty: AccountType::Base { admin: None },
+                domain: neutron_domain.clone(),
+                addr: None,
+            },
+        );
+    }
+
+    // Amount we will LP for each token
+    let lp_amount = 1_000_000u128;
+
+    info!("Inserting all services...");
+    // Reverse splitter will take tokenfactory token from account 1 and NTRN from account 2 and send it to account 3
+    workflow_config.services.insert(
+        1,
+        ServiceInfo {
+            name: "reverse_splitter".to_string(),
+            domain: neutron_domain.clone(),
+            config: ServiceConfig::ValenceReverseSplitterService(
+                valence_reverse_splitter_service::msg::ServiceConfig {
+                    output_addr: ServiceAccountType::AccountId(3),
+                    splits: vec![
+                        valence_reverse_splitter_service::msg::UncheckedSplitConfig {
+                            denom: UncheckedDenom::Native(token.clone()),
+                            account: ServiceAccountType::AccountId(1),
+                            amount:
+                                valence_reverse_splitter_service::msg::UncheckedSplitAmount::FixedAmount(
+                                    lp_amount.into(),
+                                ),
+                            factor: None,
+                        },
```

Comment:
You could use the `with_native_amount` helper function.

Reply:
tried but can't because I'm using serviceId for the workflow not Addr